test/e2e/framework/pod/wait.go

```diff
@@ -809,6 +809,18 @@ func WaitForPodContainerStarted(ctx context.Context, c clientset.Interface, name
 	})
 }
 
+// WaitForPodInitContainerStarted waits for the given Pod init container to start, after a successful run of the startupProbe.
```

Comment:
nit: Regular init containers can’t run startup probes, but restartable init containers can, so the comment isn’t entirely wrong. However, since the Sidecar containers feature is still in beta and this PR is focused on regular init containers, it might be better to remove the mention of the startup probe.

```suggestion
// WaitForPodInitContainerStarted waits for the given Pod init container to start.
```

Reply:
agree